app/src/Controller/MembersController.php

```
@@ -1333,21 +1362,27 @@ public function profilePhoto(?int $id = null): Response
     }
 
     /**
-     * Stream a member profile photo by mobile card token.
+     * Stream the authenticated member's mobile profile photo.
      *
-     * @param string|null $id Mobile card token
      * @return \Cake\Http\Response Inline file response
      */
     public function mobileCardPhoto(?string $id = null): Response
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$id'. (undefined)

(UnusedFormalParameter)